src/site/markdown/getting_started.md

```diff
@@ -1,316 +1,246 @@
+  right because types may be coerced and this could render weird results.  For instance, if using a model to predict
+  probabilities and the output type is an integral type, the values returned will likely all be 0.  This is because
+  coercion from real-valued to integrally valued numbers in done by dropping the decimal places.  If the value is in
+  the interval [0, 1), then it will be truncated to 0.
+* `--imports` adapts the DSL by importing JVM code.
+* `-p` flag says that base64-encoded protocol buffers will be used as the input format.
+
+<span class="label">bash script</span>
+
+```bash
+cat $(find $PWD/aloha-core/src -name 'fizz_buzzs.proto')               \
+| aloha-cli/bin/aloha-cli                                              \
+  -cp $(find $PWD/aloha-cli -name "*.jar" | grep dep):\
+$(find $PWD/aloha-core -name "*.jar" | grep test)                      \
+  --modelrunner                                                        \
+  --output-type Double                                                 \
+  -A                                                                   \
+  --imports "scala.math._,com.eharmony.aloha.feature.BasicFunctions._" \
+  -p com.eharmony.aloha.test.proto.Testing.UserProto                   \
+  /tmp/aloha-vw-model.json
 ```
 
-## Factory
+<span class="label label-success">output</span>
 
-Factories are important to create models: 
+<pre>
+0.732928454875946	CAESBEFsYW4YASUAALhBKg0IARABGQAAAAAAAPA/Kg0IAhACGQAAAAAAAABA
+0.7543833255767822	CAESBEthdGUYAioNCAMQAxkAAAAAAAAIQA==
+</pre>
 
-```scala
-// Scala Code
-import com.eharmony.aloha.factory.ModelFactory
-import com.eharmony.aloha.semantics.compiled.plugin.csv.CsvTypes.CsvType._
+If you don't want the input, just omit the `-A` flag or pipe and process the data elsewhere. 
 
-// vvvvv  Define the semantics  vvvvvvvvvvvvvvvvvvvvvv
+### Sanity checking the Aloha model
 
-// -----  Define the plugin  -------------------------
-val plugin = CompiledSemanticsCsvPlugin(
-  "height" -> IntType
-  "weight" -> FloatOptionType
-)
+Notice that the outputs here are:
 
-val imports = Seq("scala.math._")
-val cacheDir: java.io.File = getCacheDir() // Get a place to cache features.
-val semantics = CompiledSemantics(TwitterEvalCompiler(classCacheDir = Option(cacheDir), plugin, imports)
-// ^^^^^  Define the semantics  ^^^^^^^^^^^^^^^^^^^^^^
+* 0.732928454875946
+* 0.7543833255767822
 
-// Get a typed factory.
-val untypedFactory = ModelFactory.defaultFactory
+We saw in the section [Verifying the Model](#Verifying_the_Model), the that predictions were:
 
-val typedFactory = untypedFactory.toTypedFactory[CsvLine, Double](semantics)
+* 0.7329
+* 0.7544
 
-val modelFile: java.io.File = ...
+Which lines up.  *It appears our model is working!*
 
-// type: scala.util.Try[com.eharmony.aloha.models.Model[CsvLine, Double]]
-val modelAttempt = typedFactory.fromFile(modelFile)
-val model = modelAttempt.get 
-```
+## Programatic Aloha Model Prediction
```

Comment:
Add a note here about future support for H20.  Also mention how this could be extended to other environments (R, sklearn) easily.